.drone.yml

```diff
@@ -304,15 +304,7 @@ steps:
       - touch /root/.ssh/known_hosts
       - chmod 600 /root/.ssh/known_hosts
       - ssh-keyscan -t rsa -H github.com > /etc/ssh/ssh_known_hosts 2> /dev/null
-      - cd /infra
```

Comment:
nit: use `/infrastructure` here instead in order to avoid the ambiguity we discussed yesterday.